### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by Allstreamer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,25 @@ and this project adheres to:
...
+### Added
+
+* Added Github Actions for (cargo check, cargo fmt, cargo clippy, cargo test) [John15321](https://github.com/John15321/)
+* Added CHANGELOG.md
+* Added `ids` module
+
+### Changed
+
+* Added LICENSE with MIT License by [John15321](https://github.com/John15321/)
+* Added CI Badges to Readme by [John15321](https://github.com/John15321/)
```
Comment (John15321) [inline CHANGELOG.md]: The change itself should follow this format:
```
* Added something [#PRNUMBER](github.link/to/the/pr.com)
```

Comment (Allstreamer) [reply CHANGELOG.md]: Fixed the issue, as best i could

**Comment by Allstreamer** [comment]
Once this commits i'll look into local version stuff and consider pushing 0.2.0 to main

**Review by John15321 — APPROVED** [review]
Looks good to me

**Pull request merged**